spec/sh-options.test.sh

```diff
@@ -320,6 +320,92 @@ f.o
 .oo
 ## END
 
+#### noclobber on >>
+rm -f $TMP/no-clobber
+
+set -C
+echo foo >> $TMP/no-clobber
+echo status=$?
+
+cat $TMP/no-clobber
+## STDOUT:
+status=0
+foo
+## END
+
+#### noclobber on &> >
+set -C
+
+rm -f $TMP/no-clobber
+echo foo > $TMP/no-clobber
+echo stdout=$?
+echo bar > $TMP/no-clobber
+echo again=$?
```

Comment:
One thing I've started doing to avoid having to specify `again=1` for some shells and `again=2` for  others is

```
if test $? -ne 0; then 
  echo ok
fi
```

and then you can assert STDOUT of 'ok'

This also helps the compatibility reports at https://pages.oils.pub/

Because it means we can get more accurate assessements of other shells, without writing specific assertions for them.  If they happen to use `again=2` or `again=3` even, it is generally OK